src/ngraph/op/parameter.hpp

```diff
@@ -57,6 +58,19 @@ namespace ngraph
             bool is_relevant_to_shapes() const;
             void set_is_relevant_to_shapes(bool is_relevant);
 
+            const PartialShape& get_partial_shape() const { return m_partial_shape; }
+            PartialShape& get_partial_shape() { return m_partial_shape; }
+            void set_partial_shape(const PartialShape& partial_shape)
+            {
+                m_partial_shape = partial_shape;
+            }
+
+            const element::Type& get_element_type() const { return m_element_type; }
```

Comment:
I don't suppose it'll hurt anything, but this shadows `Node::get_element_type()`. But I guess that will eventually be going away.